seacrowd/utils/schemas/tree.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tree Schema
+
+This schema assumes a document with subnodes elements
+and a tree hierarchy.
+
+For example:
+            NODE1    .....
+        //
+ROOT    -   NODE2    .....
+        \\
+            NODE3    .....
+"""
+import datasets
```

Comment:
Hi @MJonibek, would you like to add an example of this schema using one compounded sentence on the docstring?

Reply:
Hi, ok I will do it today.

Reply:
Done. I tried to do small example, because all sentences from alt_burmese_tree_bank dataset are big. I think now it is clear how to use this schema

Reply:
Oh, so the dataset construction from the example sentence will be a top-down approach then, I thought it was a bottom-up. Looks fine to me, thanks a lot for proposing + implementing this, @MJonibek!